apps/wing-console/console/app/demo/main.w

```diff
@@ -162,7 +162,6 @@ test "Add fixtures" {
   counter.inc(100);
 }
 
-
 class WidgetService {
```

Comment:
@skyrpex I am trying this code:

```js
bring cloud;

class Foo {
  new() {
    let b = new cloud.Bucket();
  }
}

let foo = new Foo() as "foo";
for i in 0..10 {
  let bar = new Foo() as "bar-{i}";
  nodeof(foo).addConnection(source: foo, target: bar, name: "boom");
}
```

And I see a few issues:

1. I don't see the buckets
2. When I click on the connections it really show me the interaction panel for the connection.
3. The layout is weird. I expected something that is more organized...

<img width="646" alt="image" src="https://github.com/winglang/wing/assets/598796/8e54733c-97e5-477a-95e7-76feff3b3a4d">


I think it's important to make sure that *all connections are displayed*, not only the ones that connect to inflight methods!

Reply:
> I don't see the buckets

Solved the buckets issue.

> When I click on the connections it really show me the interaction panel for the connection

Not sure what you mean...

> The layout is weird. I expected something that is more organized...

From what I've seen, this is controlled by the [Node Layering Strategy](https://eclipse.dev/elk/reference/options/org-eclipse-elk-layered-layering-strategy.html). If we use `MIN_WIDTH`, it's layered mostly horizontally like your screenshot. If we use `NETWORK_SIMPLEX`, it's layered mostly vertically but it's more organized.

There are more options for the layering strategy, and some of them are "advanced" which means that they are further controlled by specific options. My take right now is to use `NETWORK_SIMPLEX`, which is more vertical'ish but feels more organized:

<img width="380" alt="image" src="https://github.com/winglang/wing/assets/1077520/af9ba2e1-0114-4874-be2a-a6f4ea1eef7c">

And then, keep tweaking all these advanced configuration options until we find what we're looking for.

Reply:
@skyrpex we can also choose some preferred layout presets and enable users to choose their favorite according to their applications using a simple dropdown

Reply:
I'll open an issue so we will be able to discuss it